index.html

```diff
@@ -65,16 +65,15 @@
           </div>
         </dl>
         <!-- @todo add tracking protection and toggle functionality -->
-        <div class="tracking-protection unimplemented">
+        <div class="tracking-protection">
           <label for="tracking-protection-control">Tracking Protection</label>
+          <label>

```

Comment:
Can this use the label above instead of using a new element here?
Something like:
```
<div class="tracking-protection">
  <label for="tracking-protection-control">
    Tracking Protection
      <div class="toggle-switch">
        <input type="checkbox" id="tracking-protection-control">
      </div>
    </label>
</div>
```
Indenting seems wrong anyway.